### PR TITLE
Updated to add an enum for snapshot access types

### PR DESCRIPTION
**Pull request by lu-c**
I was working on making changes to the request access button, and it wasn't working because it was case sensitive... so to try to address that problem, i made the value an enum.

### CODE REVIEW

File: src/pages/library/dataBrowser-utils.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ import { withErrorReporting } from 'src/libs/error'
 import { dataCatalogStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
+export const SNAPSHOT_ACCESS_TYPES = {
```
Comment (petesantos) [inline src/pages/library/dataBrowser-utils.js]: What are your thoughts on naming the top level item in camelcase? Accessing the constants would look like:
`snapshotAccessTypes.CONTROLLED`. 

We don't have a strong precedent in the codebase, but it looks like most of our enums are all camelcase.
For constants, I prefer to hold off on the capital snake case until the end as I think it looks less noisy in the code and makes it clearer when identifying the actual constant vs namespacing.

File: src/pages/library/DataBrowser.js [hunk 2]
```diff
@@ -42,33 +42,18 @@ const getUnique = (prop, data) => _.flow(
...
-          icon('lock', { style: { color: styles.access.pending, marginRight: 5 } }),
-          div(['Pending'])
-        ])
-      ]
-    }
+    labels: accessArray,
+    labelDisplays: _.zipObject(accessArray, _.map(accessKey => {
+      const lowerKey = _.toLower(accessKey)
+      return [div({ key: `access-filter-${lowerKey}`, style: { display: 'flex' } }, [
+        icon(SNAPSHOT_ACCESS_TYPES[accessKey] === SNAPSHOT_ACCESS_TYPES.OPEN ? 'unlock' : 'lock', { style: { color: styles.access[lowerKey], marginRight: 5 } }),
```
Comment (petesantos) [inline src/pages/library/DataBrowser.js]: Formatting only:
```suggestion
        icon(SNAPSHOT_ACCESS_TYPES[accessKey] === SNAPSHOT_ACCESS_TYPES.OPEN ? 'unlock' : 'lock', {
          style: { color: styles.access[lowerKey], marginRight: 5 }
        }),
```

**Review by petesantos — APPROVED** [review]
Looks good, minor nit on the casing for the constants

File: src/pages/library/dataBrowser-utils.js (hunk 1 above)
Comment (lu-c) [reply src/pages/library/dataBrowser-utils.js]: I always like using capital snake case for constants, and camel case for variables and functions, made the change tho, cause consistency in the project :)

**Review by tcjiang9 — APPROVED** [review]

**Pull request merged**